vue-frontend/src/components/contact/V2ContactForm.vue

```diff
@@ -0,0 +1,423 @@
+import axios from "axios";
+import CalendlyIframe from "./CalendlyIframe.vue";
+import config from "@/config.js";
+import loaderImage from "@/assets/images/theme/small-loader.svg";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import {
+  faPhoneVolume,
+  faEnvelope,
+  faCalendarAlt,
+  faPaperPlane,
+} from "@fortawesome/free-solid-svg-icons";
+
+const CONTACT_BY_CHAT_OR_MAIL = 1;
+
+export default {
+  data() {
+    return {
+      phone: faPhoneVolume,
+      mail: faEnvelope,
+      planeIcon: faPaperPlane,
+      calendarIcon: faCalendarAlt,
+      loaderImage: loaderImage,
+      name: "",
+      email: "",
+      projectInfo: "",
+      reference: "",
+      disableInput: false,
+      showValidationError: false,
+      openCalendlyIframeModal: false,
+      showSuccessMessagePopup: false,
+      showErrorMessagePopup: false,
+      showLoader: false,
+      contactType: 1,
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+    CalendlyIframe,
+  },
+  methods: {
+    submitApplication() {
+      if (this.name === "" || this.email === "" || this.reference === "") {
+        this.showValidationError = true;
+      } else {
+        this.disableInput = true;
+        this.showValidationError = false;
+
+        let formData = {
+          name: this.name,
+          email: this.email,
+          project_info: this.projectInfo ? this.projectInfo : "NA",
+          reference: this.reference,
+          contact_type:
+            this.contactType == CONTACT_BY_CHAT_OR_MAIL
+              ? "Chat or Email"
+              : "Call",
+        };
+
+        axios
+          .post(config.API_BASE + "/api/send-contact-mail", formData)
+          .then(() => {
+            this.showLoader = true;
+            if (this.contactType == CONTACT_BY_CHAT_OR_MAIL) {
+              setTimeout(() => {
+                this.showSuccessMessage();
+                if (this.showSuccessMessagePopup) {
+                  this.showLoader = false;
+                }
+              }, 1000);
+            } else {
+              setTimeout(() => {
+                this.openCalendlyIframe();
+                if (this.openCalendlyIframeModal) {
+                  this.showLoader = false;
+                }
+              }, 1000);
+            }
+          })
+          .catch(() => {
+            this.showErrorMessagePopup = true;
+          });
+      }
+    },
+    openCalendlyIframe() {
+      this.openCalendlyIframeModal = true;
+      this.disableInput = false;
+      this.showLoader = false;
+    },
+    closeCalendlyIframeModal() {
+      this.openCalendlyIframeModal = false;
+    },
+    showSuccessMessage() {
+      this.showSuccessMessagePopup = true;
+      setTimeout(() => {
+        this.$router.push("/");
+      }, 2000);
+    },
+  },
+};
+</script>
+
+<style lang="postcss" scoped>
+.container {
+  @apply -tw-mt-16;
+}
+
+.information-detail {
+  @apply tw-rounded-lg tw-border tw-border-slate-200 tw-bg-white tw-shadow-3xl;
```

Comment:
Increase border-radius of form

Reply:
i can't not manage that border-radius manually in tailwind css

Reply:
May be you can use `tw-rounded-xl`?